src/app/api/streaks/route.ts

```diff
@@ -0,0 +1,60 @@
+'use server';
+import { NextResponse } from 'next/server';
+import prisma from '@/lib/db';
+
+//----------------------------------------------------------------
+// this endpoint is used to get all the streaks for a user by id.
+//----------------------------------------------------------------
+export async function GET(req: Request) {
+  try {
+    // get the url from the fetch request
+    const url = new URL(req.url);
+
+    // get the skip parameter to increment the page number
+    const skip = parseInt(url.searchParams.get('skip') || '0', 10);
+
+    const userId = '1';
```

Comment:
Just making sure, this is meant to be temporary, correct?

Reply:
yes